test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -269,6 +270,23 @@ public void NgramWorkout()
             Done();
         }
 
+        [Fact]
+        void TestNgramCompatColumns()
```

Comment:
Thanks for the new test! Should we be testing the specific output values too? (or is this somewhere I'm not seeing it?)